### PR TITLE
Administrator's Features

### PR DESCRIPTION
**Pull request by ruslanshakirov**

### CODE REVIEW

File: app/assets/javascripts/admin_lists.coffee [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  switch list_value
+    when 'users_list'
+      $.ajax
+        url: "/user_list"
+    when 'ban_list'
+      $.ajax
+        url: "/ban_list"
+    when 'stories_list'
+      $.ajax
+        url: "/story_list"
```
Comment (vladimirbazhanov) [inline app/assets/javascripts/admin_lists.coffee]: you can replace all this with one line

$.ajax("#{list_value}")

File: app/controllers/admins_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AdminsController < ApplicationController
+  expose(:users) { User.not_banned }
+  expose(:stories)
+  before_action do
```
Comment (vladimirbazhanov) [inline app/controllers/admins_controller.rb]: pls ann empty line after expose and before_action

File: app/assets/javascripts/admin_lists.coffee (hunk 1 above)
Comment (ruslanshakirov) [inline app/assets/javascripts/admin_lists.coffee]: okay, thank you)

File: app/controllers/ban_list_controller.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class BanListController < ApplicationController
+  expose(:users) { User.banned }
+
+  def index
+    respond_to do |format|
+      format.js
+    end
+  end
+end
```
Comment (vladimirbazhanov) [inline app/controllers/ban_list_controller.rb]: you can add to the top of controller

respond_to :js

and all code from def index can be removed

File: app/controllers/story_list_controller.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class StoryListController < ApplicationController
+  expose(:stories)
+
+  def index
+    respond_to do |format|
+      format.js
+    end
+  end
+end
```
Comment (vladimirbazhanov) [inline app/controllers/story_list_controller.rb]: and here too

File: app/controllers/user_list_controller.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class UserListController < ApplicationController
+  expose(:users) { User.not_banned }
+
+  def index
+    respond_to do |format|
+      format.js
+    end
+  end
+end
```
Comment (vladimirbazhanov) [inline app/controllers/user_list_controller.rb]: and here

File: app/controllers/user_status_controller.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class UserStatusController < ApplicationController
+  expose_decorated(:user, attributes: :ban_param)
+  def update
+    if user.save
+      flash[:notice] = "Status has been changed"
+      redirect_to admins_path
+    end
+  end
+
+  def ban_param
```
Comment (vladimirbazhanov) [inline app/controllers/user_status_controller.rb]: this is 'user_params' not only ban)

File: app/views/user_list/index.js [hunk 7]
```diff
@@ -0,0 +1 @@
+$(".list").html("<%= escape_javascript(render partial: "user_table") %>");
```
Comment (vladimirbazhanov) [inline app/views/user_list/index.js]: escape_javascript has alias 'j' :

$(".list").html("<%= j render partial: "user_table" %>");

File: app/assets/javascripts/admin_lists.coffee (hunk 1 above)
Comment (ruslanshakirov) [inline app/assets/javascripts/admin_lists.coffee]: @vladimirbazhanov так не работает

**Pull request closed without merging**